### ISSUE TITLE
Element picker option is missing

### ISSUE THREAD

**Issue by supergunna** [uBlockOrigin/uBlock-issues#3179]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

The eyedropper icon to select the element picker option does not appear when I open the menu. The other 3 icons are there, and pressing more does not make it appear. I pressed inspect on the menu window, and it appears that the element picker button is still there somehow, but I can't click on it or see it.

### A specific URL where the issue occurs.

```yaml
https://www.wikipedia.org/
```

### Steps to Reproduce

1. Open the uBlock Origin menu by clicking on the icon in the extensions menu.

### Expected behavior

I expected to see the element picker option in the menu.

### Actual behavior

The option was not there.

### Configuration

<details>

```yaml
uBlock Origin: 1.56.0
Chromium: 123
filterset (summary):
 network: 137632
 cosmetic: 199216
 scriptlet: 29758
 html: 0
listset (total-discarded, last-updated):
 removed:
  user-filters: null
 added:
  adguard-cookies: 29009-33, 2d.2h.9m
  adguard-mobile-app-banners: 5095-8, 2d.2h.9m
  adguard-other-annoyances: 14069-10, 2d.2h.8m
  adguard-popup-overlays: 26906-83, 2d.2h.8m
  adguard-social: 22698-904, 2d.2h.8m
  adguard-widgets: 2423-3, 2d.2h.8m
  easylist-annoyances: 4437-249, 4d.1h.53m
  easylist-chat: 187-15, 4d.5h.25m
  easylist-newsletters: 7139-499, 4d.1h.52m
  easylist-notifications: 2965-1471, 4d.1h.52m
  fanboy-cookiemonster: 50386-3863, 4d.1h.52m
  fanboy-social: 15819-746, 2d.2h.7m
  [2 lists not shown]: [too many]
 default:
  easylist: 83106-75, 2h.9m Δ
  easyprivacy: 50406-121, 2h.9m Δ
  plowe-0: 3765-1184, 2d.4h.53m
  ublock-badware: 8082-146, now
  ublock-filters: 37442-697, now
  ublock-privacy: 786-7, 2h.9m Δ
  ublock-quick-fixes: 235-34, 2h.9m Δ
  ublock-unbreak: 2248-36, 2h.9m Δ
  urlhaus-1: 8705-0, 3h.9m
filterset (user): [array of 2 redacted]
trustedset:
 removed:
  chrome-extension-scheme
  chrome-scheme
switchRuleset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 2201 ms (selfie)
 maxAssetCacheWait: 385 ms
 cacheBackend: browser.storage.local
```
</details>

**Comment by stephenhawk8054** [comment]
```yml
removed:
  user-filters: null
```

You are turning off your "My filters". Turn it on again.

**Comment by garry-ut99** [comment]
> Element picker option is missing

This is not a bug, this is a feature which has been implemented recently in uBO 1.56, you most likely haven't noticed the changes introduced in uBO 1.56, as since uBO 1.56 "Element picker" is automatically disabled when "My filters" is disabled, which was not the case in previous versions of uBO, here are exaplanations / information regarding that matter :


- https://github.com/uBlockOrigin/uBlock-issues/issues/3076
- https://github.com/uBlockOrigin/uBlock-issues/issues/3161

**Comment by gorhill** [comment]
Sidenote:

>     trustedset:
>       removed:
>         chrome-extension-scheme

Removing `chrome-extension-scheme` from trusted-site directives can lead to filter update issues, mind [this important notice](https://github.com/gorhill/uBlock/wiki/Dashboard:-Trusted-sites#important-read-carefully):

> You should not remove these predefined trusted site directives, unless you know _exactly_ the consequences of doing so.
